src/routes/safe/components/Transactions/TxList/hooks/useTransactionType.ts

```diff
@@ -62,6 +62,12 @@ export const useTransactionType = (tx: Transaction): TxTypeProps => {
           break
         }
 
+        const toInfo = tx.txInfo.toInfo
+        if (toInfo) {
+          setType({ icon: toInfo.logoUri as string, text: toInfo.name })
```

Comment:
why type assertion? If we're sure it will never be `null` I think we should find a better way to identify the case.

If we're not 100% sure, doing the assertion will lead to an issue (if it is that `null` is an invalid value).

So, maybe defaulting to a valid string value would be a better solution.

```ts
setType({ icon: toInfo.logoUri ?? '', text: toInfo: name })
```

On the other hand, if null is valid, then `icon` should accept `null` as a possible value.

Reply:
agreed